quickwit-config/src/index_config.rs

```diff
@@ -127,20 +259,25 @@ mod tests {
 
     use super::*;
 
-    fn get_resource_path(relative_resource_path: &str) -> PathBuf {
-        let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
-        path.push("resources/tests/index_config/");
-        path.push(relative_resource_path);
-        path
+    fn get_resource_path(resource_filename: &str) -> String {
+        PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+            .join("resources")
```

Comment:
So I assume this is to handle windows?

On windows '/', I think '/' is still interpreted as a separator byte.
https://stdrs.dev/nightly/x86_64-pc-windows-gnu/src/std/sys/windows/path.rs.html#25-27

Reply:
Good to know, will simplify.